src/libs/style.ts

```diff
@@ -30,7 +30,12 @@ const cardStyles: Record<string, CSSProperties> = {
   },
 };
 
-export const elements = {
+export const elements: {
+  card: Record<string, CSSProperties>;
+  sectionHeader: CSSProperties;
+  pageContentContainer: CSSProperties;
+  contextBarContainer: CSSProperties;
```

Comment:
Similar to the above comment. A more concise way of representing this would be:
```ts
const elements = {
   ...
} as const satisfies Record<string, CSSProperties>;
```